model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,91 @@
+groups:
+  - id: metric.telemetry.sdk.span.ended_count
+    type: metric
+    metric_name: telemetry.sdk.span.ended_count
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.sampled_count
+    type: metric
+    metric_name: telemetry.sdk.span.sampled_count
```

Comment:
can sampled be a flag on the ended span metric? why a new metric is necessary?

Reply:
See [this comment](https://github.com/open-telemetry/semantic-conventions/pull/1631#discussion_r1863315116), I'd be glad to add a simple attribute instead